docs/tutorial/kymotracking.rst

```diff
@@ -205,21 +205,21 @@ For example::
 
     KymoWidgetGreedy(kymo, "green", axis_aspect_ratio=2, slider_ranges={"window": (0, 8)})
 
-Traced lines are accessible through the `.lines` property::
+Tracked lines are accessible through the `.lines` property::
 
     >>> lines = kymowidget.lines
-    KymoLineGroup(N=199)
+    KymoTrackGroup(N=199)
 
 For more information on its use, please see the example :ref:`cas9_kymotracking`.
 
 Using the lines algorithm
 -------------------------
 
 The second algorithm present is an algorithm that works purely on signal derivative information. It works by blurring
-the image, and then performing sub-pixel accurate line detection. It can be a bit more robust to low signal levels,
+the image, and then performing sub-pixel accurate track detection. It can be a bit more robust to low signal levels,
```

Comment:
I understand the change, but in this case, it really is detecting lines (as in the geometrical object) :/